content/articles/getting-started-with-kali-linux/index.md

```diff
@@ -0,0 +1,290 @@
+
+Nmap is a tool that gathers information by scanning the target's IP address or URL. You can learn more about Nmap scanning and interpretation from the "Introduction to Nmap" tutorial.
+
+To access Nmap, type in the `Nmap` command on the terminal followed by the target to scan as shown in the example below:
+
+```bash
+    $ map scanme.nmap.org
+```
+
+   2. **Password attacks**
+
+Tools available: `John the Ripper`, `Hydra` etc
+
+   * **John the Ripper**
+
+`John the Ripper` is a free password cracking tool originally made for Unix but now available on Kali Linux.
+It works by use of dictionary method to perform brute force on the target.
+
+You can access various tools available on John the Ripper by typing in the keywords eg `john`, `maller`, `unshadow`, `unique` etc.
+
+You can learn
+
+   3. **Reverse Engineering**
+
+Tools available: `NASM tool`, `Clang`
+
+   * **NASM tool**
+
+NASM tool allows users to program an assembler. You can do so by saving the assembly code on the cherry tree with the .asm extension. On the terminal run the commands depending on the code you are running.
+
+You can learn more on  [nasm](https://cs.lmu.edu/~ray/notes/nasmtutorial/) here.
+
+   4. **Exploitation**
+
+Tools available: `Metasploit Framework`, `Proxychains`
+
+   * **Metasploit Framework**
+
+Metasploit Framework is a powerful penetration tool that has numerous modules used to examine a system's vulnerability and possibly exploit it.
+
+You can learn more on "Metasploit Framework" in the article linked.
+
+To access Metasploit Framework on your terminal, run the `msfconsole` command and the fancy Metasploit framework pops up with `msf 5 > ` shell to run commands.
+
+```bash
+    $ msfconsole
+```
+![msfconsole](/engineering-education/getting-started-with-kali-linux/msfconsole.png)
+
+You can learn more about [metasploit framework](https://www.section.io/engineering-education/getting-started-with-metasploit-framework) in the article linked.
+
+   5. **Sniffing and Spoofing**
+
+Tools available: `Wireshark`, `Responder`
+
+   * **Wireshark**
+
+This is the world's popular network protocol analyzer. It lets users monitor what's happening on the network from a microscopic perspective.
+
+To learn more, check out [wireshark](https://tools.kali.org-informationn-gathering/wireshark) tutorial.
+
+   6. **Post Exploitation**
+
+Tools available: `Powersploit`, `Searchsploit`
+
+   * **Powerploit**
+
+This is a series of Powershell scripts that comes in handy during a post-exploitation scenario in an authorized penetration testing. They can be used in code execution, script modification, persistence, and bypass an anti-virus.
+ 
+ You can learn more about Powersploit in the article linked.
+ 
+ ### Conclusion
+ 
+ In this article, we have learned:
+ - Installing Kali Linux in a Virtual Box.
+ - Updating Kali Linux to the latest version and running some major commands.
+ - Major tools used in Kali Linux.
```

Comment:
This is more of a  summary than a conclusion. Please add a conclusive statement.